cmd/hubagent/main.go

```diff
@@ -50,7 +49,8 @@ import (
 	"github.com/kubefleet-dev/kubefleet/cmd/hubagent/workload"
 	mcv1alpha1 "github.com/kubefleet-dev/kubefleet/pkg/controllers/membercluster/v1alpha1"
 	mcv1beta1 "github.com/kubefleet-dev/kubefleet/pkg/controllers/membercluster/v1beta1"
-	fleetmetrics "github.com/kubefleet-dev/kubefleet/pkg/metrics"
+	_ "github.com/kubefleet-dev/kubefleet/pkg/metrics/hub"
```

Comment:
Instead of doing this implict call to Init function within metrics package, why not explicitly define a public function registering metrics and calling that public function within init func of main.go, wherever we need to register metrics ?